ObjectAL/ObjectAL.xcodeproj/project.pbxproj

```diff
@@ -730,6 +730,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
 				en,
 			);
```

Comment:
Do we need both `English` and `en`?

Reply:
Probably not?  I think this got added automatically by Xcode.  I'll try to see whether it's possible to remove one of them.  ✂️ Thanks for pointing this out!  👈 👍